python/lsst/ts/mtaos/mtaos.py

```diff
@@ -492,8 +496,34 @@ async def do_runWEP(self, data):
             result="runWEP started.",
         )
 
+        intra_visit_id = self.visit_id_offset + data.visitId
+        extra_visit_id = (
+            self.visit_id_offset + data.extraId if data.extraId > 0 else None
+        )
+
         if data.useOCPS:
-            raise NotImplementedError("Use OCPS not implemented.")
+            if extra_visit_id is None:
+                raise NotImplementedError("Use OCPS not implemented for LSSTCam.")
+            else:
+                config = {
+                    "LSSTComCam-FROM-OCS_DONUTPAIR": f"{intra_visit_id},{extra_visit_id}"
+                }
+
+                start_time = time.time()
+                await asyncio.create_task(
+                    self.ocps.cmd_execute.set_start(
+                        config=json.dumps(config),
+                        timeout=self.DEFAULT_TIMEOUT,
+                    )
+                )
+
+                if "RUN_WEP" not in self.execution_times:
+                    self.execution_times["RUN_WEP"] = []
+                self.execution_times["RUN_WEP"].append(time.time() - start_time)
+
+            self.model.query_ocps_results(
```

Comment:
The fact that you can fetch this from the butler is really nice. There is one caveat here though, the `execute` command will return right away, and not when the process is done. 

I think what you will have to do is pool the butler for this dataset and have a timeout for how long to wait for it. Some people might not like that we are pooling the butler so we should think of a way for RA to tell us when the process is done.